model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -49,7 +50,7 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
         ortResult.getScanResultsForId(id).flatMap {
             curationMatcher
                 .applyAll(it.summary.licenseFindings, ortResult.getLicenseFindingsCurations(id))
-                .let { findings -> FindingsMatcher().match(findings, it.summary.copyrightFindings).toSortedSet() }
+                .let { findings -> findingsMatcher.match(findings, it.summary.copyrightFindings).toSortedSet() }
```

Comment:
If the bigger topic is to avoid instance creation in "loops", this could be squashed to the previous commit with an according generic commit message. In fact I would prefer that, and you'd only need to explain the rationale once in the commit message body.

Reply:
My main reason for this change was actually readability.

Reply:
Done.